<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjustesCalculoAngulo</class>
 <widget class="QDialog" name="AjustesCalculoAngulo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>214</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajustar Calculo del Angulo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/angulo.png</normaloff>:/images/angulo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QComboBox" name="comboBoxFiltroAngulo"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelFiltroUsado">
        <property name="text">
         <string>Filtro Usado:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelAlpha">
          <property name="text">
           <string>alpha:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBoxAlphaFC">
          <property name="minimum">
           <double>0.010000000000000</double>
          </property>
          <property name="maximum">
           <double>0.990000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>0.020000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="labelTiempoCalibracion">
          <property name="text">
           <string>Tiempo Calibracion</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxTiempoCalibracion">
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSeg">
          <property name="text">
           <string>seg.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonOK">
       <property name="styleSheet">
        <string notr="true">QPushButton{
color : white;
background-color: #5cb85c;
border: 2px solid #4cae4c;
border-radius: 3px;
padding: 5px;
min-width: 80px;
}

QPushButton:hover {
background-color: #47a447;
border: 2px solid #398439;
}

QPushButton:pressed {
border: 3px solid #398439;
}</string>
       </property>
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="styleSheet">
        <string notr="true">QPushButton{
color : white;
background-color: rgb(255, 0, 0);
border: 2px solid;
border-radius: 3px;
padding: 5px;
min-width: 80px;
}

QPushButton:hover {
background-color: rgb(170, 0, 0);
border: 2px solid ;
}

QPushButton:pressed {
border: 3px solid rgb(102, 0, 0);
}</string>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
